reliability.md

```diff
@@ -1,33 +1,45 @@
 # Reliability
 
-Once your site is up, how do you keep it that way? And what can you do if it goes down? In most cases, the same best practices apply to WordPress as with other web applications, but some differences and recommendations are detailed here.
```

Comment:
In this case, I think I like the previous description a little better -- especially the latter part "In most cases, the same best practices apply to WordPress as with other web applications, but some differences and recommendations are detailed here."

I'm guessing you weren't a fan of "Once your site is up, how do you keep it that way? And what can you do if it goes down?" because it refers to you? Maybe we can rephrase that part?